packages/typography/src/FormElements.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const Mode = {
+  Light: 'light',
+  Dark: 'dark',
+} as const;
+
+type Mode = typeof Mode[keyof typeof Mode];
+
+interface ColorSets {
+  labelColor: string;
+  descriptionColor: string;
+}
+
+const colorSets: Record<Mode, ColorSets> = {
+  [Mode.Light]: {
+    labelColor: css`
+      color: ${uiColors.gray.dark2};
+    `,
+    descriptionColor: css`
+      color: ${uiColors.gray.dark1};
+    `,
+  },
+  [Mode.Dark]: {
+    labelColor: css`
+      color: ${uiColors.white};
+    `,
+    descriptionColor: css`
+      color: ${uiColors.gray.light1};
+    `,
+  },
+};
+
+const labelStyle = css`
+  font-size: 14px;
+  font-weight: bold;
+  line-height: 16px;
+  padding-bottom: 4px;
+`;
+
+type LabelProps = JSX.IntrinsicElements['label'] & {
+  darkMode?: boolean;
+  id: string | number;
+};
+
+const Label = ({ id, darkMode = false, children }: LabelProps) => {
+  const mode = darkMode ? Mode.Dark : Mode.Light;
+
+  return (
+    <label htmlFor={id} className={cx(labelStyle, colorSets[mode].labelColor)}>
```

Comment:
we're not forwarding `className` or the rest of the label props

Reply:
great call, thank you